Analytics/Classes/Crypto/SEGAES256Crypto.h

```diff
@@ -0,0 +1,23 @@
+//
+//  SEGAES256Crypto.h
+//  Analytics
+//
+//  Copyright © 2016 Segment. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "SEGCrypto.h"
+
+@interface SEGAES256Crypto : NSObject <SEGCrypto>
+
+@property (nonatomic, readonly, nonnull) NSString *password;
```

Comment:
Yes, user will need to persist password, iv, and salt in order to properly initialize the crypto object next time.